calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/SAK11204Fix.java

```diff
@@ -1,215 +0,0 @@
-/**********************************************************************************
- * $URL$
- * $Id$
- ***********************************************************************************
- *
- * Copyright (c) 2003, 2004, 2005, 2006, 2007, 2008 The Sakai Foundation
- *
- * Licensed under the Educational Community License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *       http://www.opensource.org/licenses/ECL-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- **********************************************************************************/
-
-package org.sakaiproject.calendar.impl;
-
-import java.sql.Connection;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-import java.util.Iterator;
-import java.util.List;
-
-import lombok.extern.slf4j.Slf4j;
-
-import org.sakaiproject.calendar.api.Calendar;
-import org.sakaiproject.calendar.impl.BaseCalendarService.BaseCalendarEventEdit;
-import org.sakaiproject.component.api.ServerConfigurationService;
-import org.sakaiproject.db.api.SqlService;
-
-/**
- * @author ieb
- */
-@Slf4j
-public class SAK11204Fix
-{
-	private static final int MIGRATE = 1;
-
-	private static final int UPGRADE_SCHEMA = 2;
-
-	private static final int OK = 0;
-
-	private SqlService sqlService;
-
-	private ServerConfigurationService serverConfigurationService;
-
-	private BaseCalendarService.Storage storage;
-
-	/**
-	 * @param service
-	 */
-	public SAK11204Fix(DbCalendarService service)
-	{
-		this.sqlService = service.m_sqlService;
-		this.serverConfigurationService = service.m_serverConfigurationService;
-		this.storage = service.m_storage;
-	}
-
-	/**
-	 * 
-	 */
-	public void apply(boolean autoDDL)
-	{
- 		// Check for sak11204.disable -- should only be set after successful upgrade completion
-		boolean disableUpgrade = serverConfigurationService.getBoolean("sak11204.disable", false);
```

Comment:
If we're removing this class maybe it would be good to clean up the related properties from the default file, otherwise they'll be left as orphaned forever.

Reply:
Good point.